tools/cold-backup/run.sh

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+#set -x #debug
+__TMP="/tmp/strimzi-backup" && mkdir -p $__TMP
+__HOME="" && pushd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" >/dev/null \
+    && { __HOME=$PWD; popd >/dev/null; }
+__MPOD="maintenance"
+
+SOURCE_NS="kafka"
+TARGET_NS="kafka"
+CLUSTER_NAME="my-cluster"
```

Comment:
Perhaps these should be options to the command, or should be conditionally set, so you can at least pass them like this: `CLUSTER_NAME=different-cluster run.sh`.

Reply:
Good idea and I already considered that during development. Given that there are some of them, maybe a properties file would be better.